guides-markdown/segwit.md

```diff
@@ -256,7 +262,7 @@ const txinfo = {
 const coin = Coin.fromOptions(txinfo);
 ```
 
-We use revHex to convert BE to LE.
+We use revHex to convert BE to LE ([Endianness][endianness]).
```

Comment:
Is there some information on why this conversion needs to be made? I would have had no idea this change needed to happen. I also feel like these should also be written out at least once (e.g.  `"Big Endian (BE) to Little Endian (LE)"` and then include the link), but that's just a personal preference.